src/GRA.Web/Areas/MissionControl/Views/Drawing/CriteriaDetail.cshtml

```diff
@@ -1,16 +1,21 @@
 ﻿@model GRA.Controllers.ViewModel.MissionControl.Drawing.CriterionDetailViewModel
 
-    <div style="font-size:large;  padding-top:10px;">
-        Created By:
-        @if (Model.CanViewParticipants)
-        {
-            <a asp-controller="Participants" asp-action="@nameof(ParticipantsController.Detail)" asp-route-id="@Model.Criterion.CreatedBy" target="_blank">@Model.CreatedByName</a>
-        }
-        else
-        {
-            @Model.CreatedByName
-        }
-    </div>
+@inject Microsoft.Extensions.Localization.IStringLocalizer<GRA.Resources.Shared> SharedLocalizer
```

Comment:
Localization isn't needed on the MissionControl side of things.

Reply:
Fixed in commit a644d762f46ed05c11f0f643e52b135003a8f354